content/pages/irc.md

```diff
@@ -18,9 +18,9 @@ we do not publish our logs. Communication in the channel is governed by our
 [Code of Conduct]({filename}/pages/code-of-conduct.md). If you haven't given it
 a read yet, please take a few minutes to familiarize yourself with it!
 
-# Installation instructions #
+## Setting Up IRC (you can pick any of these options)
```

Comment:
I think "Use whichever option works best for you" would be better on the line below the title. It's an instruction, so it doesn't super fit being in the header.